src/main/java/com/dsrm/dsrmbackend/controllers/UserController.java

```diff
@@ -46,8 +46,12 @@ ResponseEntity<UserDTO> getUser(@PathVariable Long id){
 
     @GetMapping(value = "/users")
     @PageableAsQueryParam
-    public ResponseEntity<Page<UserDTO>> readUsers(@Parameter(hidden = true) Pageable pageable) {
-        Page<User> userPage = userService.getUsers(pageable);
+    public ResponseEntity<Page<UserDTO>> readUsers(@Parameter(hidden = true) Pageable pageable,@RequestParam(required = false,defaultValue = "false") boolean isPending) {
+        Page<User> userPage;
+        if (isPending)
```

Comment:
we don't want to have any logic in controller. Move this if to service and pass isPending param